### ISSUE TITLE
the proxy(-p) issuei during install

### ISSUE THREAD

**Issue by CNplasma** [dnschneid/crouton#4784]
crosh> shell
chronos@localhost / $ sudo crouton -u -p http://127.0.0.1
Downloading latest crouton installer...
###################################################################################################################### 100.0%
/http:/127.0.0.1 is not a valid prefix
chronos@localhost / $ sudo crouton -t gnome -p http://127.0.0.1
/http:/127.0.0.1 is not a valid prefix

When i need to use proxy to upadate, It just pop up like this, no matter how i changed the address, it just no work.
So how exactly the "vaild prefix" be like, thanks for your help.

**Comment by TomTravis** [comment]
yes the -p in this case is Path not internet proxy for a device IE /media/removable/sdcard/dirname
when enter-chroot is -c  /media/removable/sdcard/dirname/chroots/ -n nameofchroot
